hpi-swt2/lecture-portal
# Add boilerplate to use React, TypeScript and Webpacker

This PR adds Webpacker, React and TypeScript to the project.

Testing steps:
- start the application and you should see `Hello World from Webpacker` on the web console

Acceptance Criteria:
- you can make use of React, TypeScript and Webpacker

Solves issue #26 

Steps needed for migration:
- Install newest nodejs and yarn version
